.yarnrc

```diff
@@ -0,0 +1 @@
+--install.no-lockfile true
```

Comment:
not sure to understand why we should do this
it's bad practice to have two different package manager for the same project, either we use `yarn` (and we drop the `npm-.*` files), either we use `npm` and we drop `yarn.lock`

(in both cases, that doesn't seem related to this PR)

Reply:
Done.

Successfully tested locally. I am a little bit unsure about the Github Actions (particularly the `key` for `actions/cache@v2`), though.